### ISSUE TITLE
[BUG]Emit PartitionId in gateway metrics

### ISSUE THREAD

**Issue by xinlian12** [Azure/azure-sdk-for-java#39516]
adding PartitionId dimension for gateway statistics

**Comment by Injinius** [comment]
@xinlian12 This ticket is almost a year old. Is the issue resolved? 

If not, then can you provide a more detailed description of the issue you are facing. Based on the ticket's labels, it appears you are interested in cosmos metrics. I do see see a "PartitionId dimension" on the following [page](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/cosmos/azure-cosmos/docs/Metrics.md#which-tagsdimensions-are-associated-with-metrics). If that is related to the "PartitionId" in your title, then please provide details/examples on what you believe is the associated bug and what the desired solution would look like..

**Comment by FabianMeiswinkel** [comment]
Yes, the ask is to capture the Physical Partition Id as a dimension in Cosmso DB metrics emitted. I think it is addressed in the emantime - but needs to be re-validated.

**Comment by Injinius** [comment]
In the interests of cleaning up "old" tickets, can this one be closed? Maybe create a new one to handle the "re-validation" work?

**Comment by kushagraThapar** [comment]
Closing this because of inactivity, please reopen if the issue still persists.
